src/ecs/commands/runCommandInContainer.ts

```diff
@@ -68,7 +63,8 @@ export async function runCommandInContainer(
             outputChannel
         )
 
-        const cp = await new ChildProcess(ssmPlugin, args, { logging: 'noparams' }).run({
+        await new ChildProcess(ssmPlugin, args, { logging: 'noparams' }).run({
+            rejectOnExit: true,
```

Comment:
Would `rejectOnNonZeroExit` be too verbose?
Or we could make rejection be the default.

Reply:
`rejectOnErrorCode`? Having this as non-default seems fine to me, sometimes you might want to capture the exit code.

Reply:
`rejectOnFail` or `rejectOnErrorCode` seem better